h2o-genmodel/src/main/java/hex/genmodel/ModelMojoReader.java

```diff
@@ -130,7 +149,7 @@ else if (line.equals("[columns]")) {
       } else if (section == 1) {
         // [info] section: just parse key-value pairs and store them into the `info` map.
         String[] res = line.split("\\s*=\\s*", 2);
-        info.put(res[0], res[0].equals("uuid")? res[1] : ParseUtils.tryParse(res[1]));
+        info.put(res[0], res[0].equals("uuid")? res[1] : new RawValue(res[1]));
```

Comment:
The reason why I added the RawValue is because the _lkv map hold both parsed values (domains, n_cols... perhaps others) and raw (unparsed) values, I needed an marker class to distinguish the unparsed strings from values already parsed as strings (eg. UUIDs seems to have special handling).

I think we will need to revisit the implementation on the MojoReader in the future. Are you okay with keeping it as is for now?